src/plugin/action/join.sjs

```diff
@@ -104,10 +104,16 @@ module.exports = function (client, rawf, emitter) {
             };
 
             client.debug("PluginAction", formatc("Attempting to join %s."));
```

Comment:
This will print "[Object object]" if type is object. As such, this debug needs to be inside the else case, and a different debug needs to be inside the passworded version.